bool new_trip
string service_station_id
